### DISCUSSION TITLE
Attachable property "Resources" not found in type Window

### DISCUSSION THREAD

**Discussion by lavinders** [microsoft/WindowsAppSDK#1974] [Q&A] [answered]
Hi, i created a new packaged WinUI project and simply adding <Windows.Resources>.. tag in XAML shows an error, anybody knows how to fix it? I don't see this error if i create a new page and add Page.Resources however i am little confused by seeing this tag in many other xaml projects.

```
<Window
    x:Class="App3.MainWindow"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="using:App3"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    mc:Ignorable="d">

    <Window.Resources>
        
    </Window.Resources>
    
    
    <StackPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Center">
        <Button x:Name="myButton" Click="myButton_Click">Click Me</Button>
    </StackPanel>
</Window>
```

**Comment by kmgallahan** [comment, marked as answer]
https://learn.microsoft.com/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.page?#properties

https://learn.microsoft.com//windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.window?#properties

Compare and contrast.

**Reply by lavinders** [reply]
Thank you, i was confused by previous XAML types where Window seems to have Resources property.

**Reply by ted537** [reply]
Links are now dead :(

**Reply by DarranRowe** [reply]
They are easy enough to find through a search engine, but the current link to the Windows App SDK namespaces is [here](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/). You can easily get to the above runtime classes from there.

**Reply by kmgallahan** [reply]
I updated the links.
